packages/cli-build/test/wait.test.js

```diff
@@ -133,6 +133,22 @@ describe('percy build:wait', () => {
     ]));
   });
 
+  it('does not error on diffs if the review status is approved and pass-if-approved is on', async () => {
```

Comment:
Please add additional coverage with 
* `--pass-if-approved` passed `--fail-on-changes` not passed, and vice versa
* when both are passed.

Reply:
done!